libraries/Wire/src/Wire.cpp

```diff
@@ -338,10 +340,12 @@ bool TwoWire::end()
         }
 #endif
         if(is_slave){
+#if SOC_I2C_SUPPORT_SLAVE
```

Comment:
why only this code and initializations above commented? If this does not cause issues, please leave it alone and add a note in the PR for me to implement it properly.

Reply:
removed. Thx for looking into doing a real fix :-)

Reply:
I guess we just haven't had a chip without it yet :) no worries

Reply:
https://github.com/espressif/arduino-esp32/pull/8882 created. Will merge it after CI and pull master here.